src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -57,16 +62,30 @@ public SearchService(BigQueryPdao bigQueryPdao, RestHighLevelClient client) {
         this.client = client;
     }
 
+    private String uuidToIndexName(UUID id) {
+        return String.format("idx-%s", id);
+    }
+
+    private String translateToTim(String sql) {
+        Pattern regex = Pattern.compile("( as )(\\w+)");
+        Matcher matches = regex.matcher(sql.toLowerCase());
```

Comment:
Does this result in case folding of the column name before it's encoded? If the intent is to case fold `as` I think you'd be better off using a regex like `( [aA][sS] )`.

Reply:
I'm not sure what the best approach is here. I had thought of using `[as|AS]` but we control the inputs and column names, and lowercasing seems like a reasonable way to normalize inputs. Perhaps I should only lowercase the field names and not the entire query?

Reply:
I decided to not normalize at all, that way only the columns get modified.